app/services/GetPenaltyDetailsService.scala

```diff
@@ -51,4 +54,29 @@ class GetPenaltyDetailsService @Inject()(getPenaltyDetailsConnector: GetPenaltyD
         res
     }
   }
+
+  private def filterEsyimatedLPP1(penaltiesDetails: GetPenaltyDetails, vrn: String): GetPenaltyDetails = {
+    if (penaltiesDetails.latePaymentPenalty.nonEmpty) {
+      val filteredLPPs: Option[Seq[LPPDetails]] = penaltiesDetails.latePaymentPenalty.flatMap(
+        _.details.map(latePaymentPenalties => latePaymentPenalties.filterNot(lpp =>
+          lpp.penaltyCategory.equals(LPPPenaltyCategoryEnum.FirstPenalty) &&
+            lpp.penaltyStatus.equals(LPPPenaltyStatusEnum.Accruing) &&
+            appConfig.withinLPP1FilterWindow(lpp.principalChargeDueDate))
+        )
+      )
+
+      if (filteredLPPs.nonEmpty) {
+        val numberOfFilteredLPPs: Int = filteredLPPs.get.size - penaltiesDetails.latePaymentPenalty.get.details.size
+        logger.info(s"[GetPenaltyDetailsService][filterEsyimatedLPP1] - Filtered $numberOfFilteredLPPs LPP1(s) from payload for VRN: $vrn")
```

Comment:
Maybe worth clarifying that the LPP1's that were filtered were estimates. I know the method is filterEstimatedLPP1 however just to make it super clear:
```suggestion
        logger.info(s"[GetPenaltyDetailsService][filterEstimatedLPP1] - Filtered $numberOfFilteredLPPs estimated LPP1(s) from payload for VRN: $vrn")
```

Reply:
Fixed ✅